resource-aggregate/events/deviceMetadataSnapshotTaken.go

```diff
@@ -131,25 +131,19 @@ func (e *DeviceMetadataSnapshotTaken) Handle(ctx context.Context, iter eventstor
 			if err := eu.Unmarshal(&s); err != nil {
 				return status.Errorf(codes.Internal, "%v", err)
 			}
-			if err := e.HandleDeviceMetadataSnapshotTaken(ctx, &s); err != nil {
-				return err
-			}
+			_ = e.HandleDeviceMetadataSnapshotTaken(ctx, &s)
```

Comment:
Feels weird to just ignore an error, wouldn't at least logging it be useful?

Reply:
Removed error returns for snapshot handlers.